src/app/fdctl/mem.c

```diff
@@ -1,10 +1,11 @@
 #include "fdctl.h"
 
+#include "run/run.h"
+
 void
 mem_cmd_fn( args_t *         args,
             config_t * const config ) {
   (void)args;
 
-  fd_topo_fill( &config->topo, FD_TOPO_FILL_MODE_FOOTPRINT );
```

Comment:
I'm not sure why these aren't needed anymore, but I'll trust you on it. It would have failed pretty quickly if they were still needed.

Reply:
I moved the footprint fill into config.c when it creates the topology. This will change a bit more in another incoming change.